student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(name, age, parent_permission: true, classroom: nil)
+    super(name: name, age: age, parent_permission: parent_permission)
```

Comment:
- [ ] `classroom` is not an optional parameter. Kindly fix that. The `name` should be optional and the parameters for the super method are wrongly set up.

  ```
  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
  ```